modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -23,42 +23,43 @@ installation program created to your HTTP server. Note the URLs of these files.
 +
 [IMPORTANT]
 ====
+You can add or change configuration settings in your Ignition configs
+before saving them to your HTTP server.
 If you plan to add more compute machines to your cluster after you finish
 installation, do not delete these files.
 ====
 
 ifndef::openshift-origin[]
-. Obtain the {op-system} ISO image, compressed metal RAW image, `kernel`
-and `initramfs` files from the
+. Obtain the {op-system} `kernel`,
+`initramfs` and `rootfs` files from the
 link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red
 Hat customer portal or the
 link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.5/[{op-system} image mirror]
 page.
 +
 [IMPORTANT]
 ====
-The {op-system} images might not change with every release of {product-title}.
-You must download images with the highest version that is less than or equal
-to the {product-title} version that you install. Use the image versions
-that match your {product-title} version if they are available.
-Only use ISO images for this procedure.
+The {op-system} artifacts might not change with every release of {product-title}.
+You must download artifacts with the highest version that is less than or equal
+to the {product-title} version that you install. Only use
+the appropriate kernel, initramfs, and rootfs artifacts described below
```

Comment:
initramfs and rootfs should be in back ticks

Reply:
Done.